app/Models/Seat.php

```diff
@@ -9,6 +9,7 @@ class Seat extends Model
     protected $fillable = [
         'name',
         'location_id',
+        'user_id'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoCommaAfterLast</br>**Severity**: error</br>**File**: app/Models/Seat.php</br>**Line**: 12</br>Comma required after last value in array declaration